From 2a4ba77b573a1e60542d5b2d19b62908bf167e4f Mon Sep 17 00:00:00 2001
From: Bopamo Osaisai <bopamo@gmail.com>
Date: Fri, 8 Jan 2016 09:15:31 -0800
Subject: [PATCH 06/88] Staging: android: Fix brace coding style warning in
 sync_debug.c
Git-commit: cd572182bd3d82e270d273b241f29d40b4213111
Patch-mainline: v4.6-rc1
References: fate#321223

This is a patch to the sync_debug.c file that rectifies a brace warning
that was found with the checkpatch.pl tool

Signed-off-by: Bopamo Osaisai <bopamo@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Patrik Jakobsson <pjakobsson@suse.de>
---
 drivers/staging/android/sync_debug.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/staging/android/sync_debug.c b/drivers/staging/android/sync_debug.c
index f45d13cdd42b..02a1649a1bd9 100644
--- a/drivers/staging/android/sync_debug.c
+++ b/drivers/staging/android/sync_debug.c
@@ -158,9 +158,8 @@ static void sync_print_fence(struct seq_file *s, struct sync_fence *fence)
 	seq_printf(s, "[%p] %s: %s\n", fence, fence->name,
 		   sync_status_str(atomic_read(&fence->status)));
 
-	for (i = 0; i < fence->num_fences; ++i) {
+	for (i = 0; i < fence->num_fences; ++i)
 		sync_print_pt(s, fence->cbs[i].sync_pt, true);
-	}
 
 	spin_lock_irqsave(&fence->wq.lock, flags);
 	list_for_each_entry(pos, &fence->wq.task_list, task_list) {
-- 
2.12.0
